WeScan/Review/ReviewViewController.swift

```diff
@@ -55,24 +57,29 @@ final class ReviewViewController: UIViewController {
         super.init(nibName: nil, bundle: nil)
     }
     
-    required init?(coder aDecoder: NSCoder) {
+    required public init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
     
-    override func viewDidLoad() {
+    override open func viewDidLoad() {
```

Comment:
My bad if I'm wrong - but if we decide to make the view controllers open, don't methods inherit permissions?